### PR TITLE
chore(docs): migrate react-accordions examples to storybook

### PR DESCRIPTION
**Pull request by austin94**
## Description

This PR migrates the `Stepper` and `Accordion` examples from styleguidist to storybook.

## Detail

This one of the first "complex" examples that is being brought over. I've created some custom `argTypes` for the story controls to match our knobs example. Let me know what you think.

## Checklist

- [ ] ~:ok_hand: design updates are Garden Designer approved (add the
      designer as a reviewer)~
- [x] :globe_with_meridians: Styleguidist demo is up-to-date (`yarn start`)
- [x] :arrow_left: renders as expected with reversed (RTL) direction
- [x] :metal: renders as expected with Bedrock CSS (`?bedrock`)
- [x] :wheelchair: analyzed via [axe](https://www.deque.com/axe/) and evaluated using VoiceOver
- [ ] ~:guardsman: includes new unit tests~
- [x] :memo: tested in Chrome, Firefox, Safari, Edge, and IE11

### CODE REVIEW

File: packages/accordions/stories/2-Accordion.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,320 @@
+/**
```
Comment (hzhu) [inline packages/accordions/stories/2-Accordion.stories.tsx:1]: Is there a particular reason files under `/accordions/stories` are prefixed with a number like `1-filename.ext`? Should files under `/tabs/stories` follow a number prefix naming?

**Review by hzhu — APPROVED** [review]

File: packages/accordions/stories/2-Accordion.stories.tsx (hunk 1 above)
Comment (austin94) [reply packages/accordions/stories/2-Accordion.stories.tsx:1]: This is to ensure the correct ordering of stories:
- `README` first
- Alphabetical groups after that

I've seen this method used in other storybooks, but if there is a cleaner way I would like to get rid of these.

File: packages/accordions/stories/2-Accordion.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+Advanced.args = {
+  isCompact: false,
+  isBare: false
+};
+
+Advanced.parameters = {
+  docs: {
+    description: {
+      story: `
+  The \`Accordion\` component is implemented using the [W3C Accordion pattern](https://www.w3.org/TR/wai-aria-practices/#accordion).
```
Comment (jzempel) [inline packages/accordions/stories/2-Accordion.stories.tsx:310]: This repeat from the section above might not be necessary.

**Review by jzempel — COMMENTED** [review]
What do we think about renaming the sections under accordion to "Uncontrolled" and "Controlled"?

File: packages/accordions/stories/2-Accordion.stories.tsx (hunk 2 above)
Comment (austin94) [reply packages/accordions/stories/2-Accordion.stories.tsx:310]: Removed.

**Comment by austin94** [comment]
@jzempel I've renamed these to "Uncontrolled" and "Controlled". I also split the examples into two separate files to help with readability.

**Review by jzempel — APPROVED** [review]
purdy.

File: packages/accordions/stories/examples/Controlled.tsx [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+Controlled.args = {
+  isCompact: false,
+  isBare: false
+};
+
+Controlled.parameters = {
+  docs: {
+    description: {
+      story: `
+  #### Controlled usage
```
Comment (jzempel) [inline packages/accordions/stories/examples/Controlled.tsx:199]: [nit] this line might be superfluous.

Comment (austin94) [reply packages/accordions/stories/examples/Controlled.tsx]: Removed.

**Pull request merged**